project/project-info.conf

```diff
@@ -92,6 +92,16 @@ project-info {
       }
     ]
   }
+  akka-discovery-azure-api: ${project-info.shared-info} {
+    title: "Akka Discovery Azure"
+    levels: [
+      {
+        readiness: Supported
+        since: "2024-08-15"
+        since-version: "1.5.3"
```

Comment:
I'm not sure if I got this version right. 🤔

Reply:
current version is 1.5.3, so this should be 1.5.4